curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-destructuring-assignment-with-the-rest-parameter-to-reassign-array-elements.md

```diff
@@ -64,23 +64,25 @@ assert(
 ## --seed-contents--
 
 ```js
-const source = [1,2,3,4,5,6,7,8,9,10];
 function removeFirstTwo(list) {
   // Only change code below this line
   const arr = list; // Change this line
   // Only change code above this line
   return arr;
 }
+
+const source = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 const arr = removeFirstTwo(source);
```

Comment:
Do we want to use more descriptive names here as well? Perhaps `original` and `updated` or something? I generally dislike calling an array `arr` because that only communicates the type, not the purpose.

Reply:
I'm a fan of unnecessarily explicit variable names - maybe `source` and `sourceWithFirstTwoRemoved`?

Definitely agree that `arr` is not an ideal variable name, though.